src/java/fr/paris/lutece/plugins/suggest/web/SuggestApp.java

```diff
@@ -1186,7 +1187,7 @@ private void doVote( String strVote, int nIdSubmitSuggest, Plugin plugin, String
         // Increment vote
         int nScore;
 
-        if ( ( strVote != null ) && strVote.matches( ( "-2|-1|1|2" ) ) )
+        if ( ( strVote != null ) && strVote.matches( DO_VOTE_CASES ) )
```

Comment:
This does not change the fact that the regexpr is compiled on each invocation. See https://github.com/lutece-platform/lutece-chat-plugin-easyrulesbot/blob/ab7f68990a4568c9bcca9737175a67f5cee0dc5a/src/java/fr/paris/lutece/plugins/easyrulesbot/service/response/processors/InputStringResponseProcessor.java#L50 for an example

Reply:
Thank you.